solidity/contracts/token/README.md

```diff
@@ -39,7 +39,7 @@ graph LR
         Mailbox_G[(Mailbox)]
     end
 
-    HYP_E -. "router" .- HYP_P -. "router" .- HYP_G
+    HYP_E -. "TokenMessage" .- HYP_P -. "TokenMessage" .- HYP_G
```

Comment:
this was supposed to indicate that the token routers are aware of each other's addresses via the router table
it is also true that they communicate with TokenMessage (via the Mailbox)